org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+      - type: git
+        url: 'https://gitlab.gnome.org/GNOME/libgee.git'
+        #branch: master
+        tag: '0.20.6'
+    post-install:
+      - ln -s /app/share/vala/vapi/gee-0.8.vapi /usr/lib/sdk/vala/share/vala-0.56/vapi/
+      - ln -s /app/share/gir-1.0/Gee-0.8.gir /usr/share/gir-1.0/
+    cleanup:
+      - /include
+      - /share
+      - /lib/pkgconfig
+
+  #-------------------------------------------------------------------------------
+  # wxFormBuilder
+  #-------------------------------------------------------------------------------
+  - name: wxformbuilder
+    sources:
+      - type: git
+        url: 'https://github.com/wxFormBuilder/wxFormBuilder'
+        #branch: master
+        tag: v3.9.0
+    buildsystem: simple
+    config-opts:
+      - --disable-mediactrl
+    build-commands:
+      - ./create_build_files4.sh
+      - sed -i "44 a \#include <cstdint>" src/md5/md5.hh
+      - make config=release -C build/3.0/gmake
+      - mv output/bin/wxformbuilder /app/bin/
+      - mkdir /app/share/wxformbuilder/
+      - mv output/xml/ /app/share/wxformbuilder/
+      - mv output/resources/ /app/share/wxformbuilder/
+      - mv output/plugins/ /app/share/wxformbuilder/
+      - mv output/*.* /app/share/wxformbuilder/
+      - mv output/lib/* /app/lib/
+      - mkdir /app/share/applications/
+      - mv install/linux/data/gnome/usr/share/applications/wxformbuilder.desktop /app/share/applications/org.wxformbuilder.wxFormBuilder.desktop
+      - sed -i 's|wxformbuilder.png|org.wxformbuilder.wxFormBuilder.png|' /app/share/applications/org.wxformbuilder.wxFormBuilder.desktop
+      - mkdir -p /app/share/icons/
+      - mv install/linux/data/gnome/usr/share/pixmaps/wxformbuilder.png /app/share/icons/org.wxformbuilder.wxFormBuilder.png
+      - mkdir /app/share/appdata/
+      - mv install/linux/data/gnome/usr/share/appdata/org.wxformbuilder.wxFormBuilder.appdata.xml /app/share/appdata/
+    cleanup:
+      - '*'
+
+  #-------------------------------------------------------------------------------
+  # yat
+  #-------------------------------------------------------------------------------
+  - name: yat
+    sources:
+      - type: dir
+        path: ./
+        skip:
+          - .flatpak-builder/
+          - build/
+          - org.gitlab.neva_blyad.yat.yml
+          - python3-scons.json
+    buildsystem: simple
+    build-options:
+      env:
+        - ARCH=x86_64-linux-gnu # TODO: do cross-platform
+        - PKG_CONFIG_PATH=/app/lib/pkgconfig/:/app/lib/x86_64-linux-gnu/pkgconfig/ # TODO: do cross-platform
+    build-commands:
+      #- ln -s /usr/lib/sdk/vala/share/vala/vapi/gee-0.8.vapi /usr/lib/sdk/vala/share/vala-0.56/vapi/
+      - ln -s /app/lib/libwx_baseu_xml-3.0.so   /usr/lib/x86_64-linux-gnu/
+      - ln -s /app/lib/libwx_baseu_xml-3.0.so.0 /usr/lib/x86_64-linux-gnu/
+      - ln -s /app/lib/libwx_baseu-3.0.so       /usr/lib/x86_64-linux-gnu/
+      - ln -s /app/lib/libwx_baseu-3.0.so.0     /usr/lib/x86_64-linux-gnu/
```

Comment:
I don't see why this would be needed, beside not working since `/usr` is read only.

Reply:
Sadly, my app doesn't link with wxWidgets without these links

Reply:
kk